### ISSUE TITLE
A recent glossary entry I added is shown listed as "unfinished" and "untranslated"

### ISSUE THREAD

**Issue by yilmazdurmaz** [WeblateOrg/weblate#9629]
### Describe the issue

I added a new entry "Buffer" to the glossary with its translation "AraBellek", but now it is listed as "unfinished" and "untranslated"

main: https://hosted.weblate.org/projects/godot-engine/glossary/tr/
unfinished: https://hosted.weblate.org/browse/godot-engine/glossary/tr/?q=state:empty
untranslated: https://hosted.weblate.org/browse/godot-engine/glossary/tr/?q=state:%3Ctranslated

can you please check for the problem?

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

I don't know how to reproduce. It just happened.

### Expected behavior

This string has its translation, so it should not be listed as such.

### How do you run Weblate?

weblate.org service

**Comment by nijel** [comment]
Can you please describe how did you add the glossary entry?

**Comment by yilmazdurmaz** [comment]
I use two variations to add a new glossary item:
- while translating a string ("translate/project_name/component_name/lang_code"), click on "add term to glossary" button in the glossary panel*, fill in a string and its translation in the modal*, press "Enter"
- same as above, but use "Ctrl+Enter". but this does not seem to differ. I tried using shift, alt and combinations, nothing seems changed.

I traversed glosssary once and corrected mistakes previously added, and almost never added a new string while in the overview of the glossary page ("projects/project_name/glossary"), nor while in whole list of glossary items ("browse/project_name/glossary").

I had once added a term with all its tools* checked (terminology, forbidden, untanslatable) without actually clicking on those checkboxes, but that was an accident for me and I don't know how to do that again. (maybe another bug!?)

for these reasons, I have no idea how this string got into this state.
